star-ui/src/App.tsx

```
@@ -7,44 +9,195 @@ import { OdometryPanel } from './components/OdometryPanel';
+              </div>
+            ))}
+          </ResponsiveGridLayout>
+        )}
       </div>
+
+      {/* Maximized Panel Overlay */}
+      {maximizedPanel && panelRegistry[maximizedPanel] && (
+        <div
+          className="panel-maximize-overlay"
+          onClick={(e) => { if (e.target === e.currentTarget) toggleMaximize(maximizedPanel); }}
+        >
```

Biome (2.4.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

(lint/a11y/useKeyWithClickEvents)